providers/implementations/kdfs/srtpkdf.inc.in

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2025 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the \"License\").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+{-
+use OpenSSL::paramnames qw(produce_param_decoder);
+-}
+
+{- produce_param_decoder('srtp_set_ctx_params',
+                         (['OSSL_KDF_PARAM_PROPERTIES',    'propq',  'utf8_string'],
+                          ['OSSL_KDF_PARAM_CIPHER',        'cipher', 'utf8_string'],
+                          ['OSSL_KDF_PARAM_KEY',           'key',    'octet_string'],
+                          ['OSSL_KDF_PARAM_SALT',          'salt',   'octet_string'],
+                          ['OSSL_KDF_PARAM_SRTPKDF_KDR',   'kdr',    'octet_string'],
```

Comment:
kdr should just be an int.
Its value is 0...800000 (so you can test that the value lies in that range)
It has the restriction that only 1 bit is set. This is easy to test in a loop and also find the power of 2 required to do a shift (i.e. divide by)